test/e2e/storage/drivers/in_tree.go

```diff
@@ -61,8 +61,6 @@ import (
 
 // NFS
 type nfsDriver struct {
-	serverIP               string
```

Comment:
@mkimuram was there some dependency between nfs server pod and external provisioner?

Reply:
I think drivers that store their state in Pods are ok because it gets cleaned up by e2e framework deleting the test namespace at the end.